package.json

```diff
@@ -16,13 +16,18 @@
     "start": "next start",
     "make-seo": "node make-seo.js",
     "schedules": "node schedules.js",
-    "lint": "npx eslint . --fix --ext .ts,.tsx"
+    "lint": "npx eslint . --fix --ext .ts,.tsx",
+    "deploy:local": "node ./scripts/deploy -n development",
```

Comment:
and if we can have the `script/deploy.js` receive the rpcUrl and all the needed arguments then we don't need to have all these scripts (nor keep adding more) because we can simply run a command :D

Reply:
it is hardhat like script. I used the same approach than other projects like uniswap . you dont have to remember the RPC . Just type the command and the rest is magic.